obs/project.go

```diff
@@ -76,8 +78,8 @@ const (
 
 type Repository struct {
 	Repository     string           `json:"name" xml:"name,attr"`
-	Architectures  []string         `json:"arch" xml:"arch"`
-	ReleaseTargets []ReleaseTarget  `json:"releasetarget,omitempty" xml:"releasetarget,omitempty"`
+	Architectures  []string         `json:"architectures" xml:"arch"`
+	ReleaseTargets []ReleaseTarget  `json:"releaseTarget,omitempty" xml:"releasetarget,omitempty"`
```

Comment:
@cpanato I proposed this change in the previous PR. JSON works in a bit different way than XML. For example, you **don't** have arrays in XML, but you have arrays in JSON. Because of that, I would prefer that JSON tags are called same as fields, i.e. plural and camel-case. In my opinion, in the JSON world, `architectures` is more natural than `arch`